src/main/java/com/app/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package com.app.bookstore.dto;
 
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;

```

Comment:
ISBN has other business requirements (format and fixed length). So validate it too.